# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/tcsh/undef_tiocgltc.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


This patch is needed on Alpha AXP ...

--- ./ed.h.orig	Sat Nov 25 09:42:59 2000
+++ ./ed.h	Sat Nov 25 09:43:03 2000
@@ -37,6 +37,10 @@
 #ifndef _h_ed
 #define _h_ed
 
+#ifdef TIOCGLTC
+# undef TIOCGLTC
+#endif
+
 #ifndef EXTERN
 # define EXTERN extern
 #endif
